README.rst

```diff
@@ -72,17 +72,14 @@ When this library reaches v1.0.0, we will aim to maintain backwards-compatibilit
 About
 -----
 
-This SDK aims to provide an easy-to-use framework for managing a pi-top. It includes a Python 3 package (`pitop`),
-with several modules for interfacing with a range of pi-top devices and peripherals It also contains CLI utilities,
+This SDK aims to provide an easy-to-use framework for managing a pi-top. It includes several Python 3 packages and
+modules for interfacing with a range of pi-top devices and peripherals It also contains CLI utilities,
 to interact with your pi-top using the terminal.
```

Comment:
I know this wasn't introduced by this PR but would be good to get it in at the same time. :)
```suggestion
modules for interfacing with a range of pi-top devices and peripherals.
It also contains CLI utilities for interacting with your pi-top using the terminal.
```